.travis.yml

```diff
@@ -8,6 +8,11 @@ install:
   - pip3 install --user -r requirements.txt
   - curl -sfL https://raw.githubusercontent.com/securego/gosec/master/install.sh | sh -s -- -b $GOPATH/bin 1.2.0
   - export GOPATH=$GOPATH:${TRAVIS_BUILD_DIR}/test/fixtures/go
+  - npm install -g codecov
+
+script:
+  - jest && codecov
```

Comment:
The default for script on node_js is `yarn test`.  Now you are switching to `jest`.